apps/src/music/views/MusicPlayView.tsx

```diff
@@ -3,25 +3,37 @@ import React, {memo, useCallback, useMemo} from 'react';
 import {Button} from '@cdo/apps/componentLibrary/button';
 import {Heading2, BodyTwoText} from '@cdo/apps/componentLibrary/typography';
 import commonI18n from '@cdo/locale';
-import codeOrgLogo from '@cdo/static/code-dot-org-white-logo.svg';
+import musicPlayViewLogo from '@cdo/static/music/music-play-view.png';
 
 import moduleStyles from './music-play-view.module.scss';
 
 import musicI18n from '../locale';
+import {useAppSelector} from '@cdo/apps/util/reduxHooks';
+import {queryParams} from '@cdo/apps/code-studio/utils';
+import Lab2Registry from '@cdo/apps/lab2/Lab2Registry';
 
 interface MusicPlayViewProps {
-  isPlaying: boolean;
-  onPlay: () => void;
-  onStop: () => void;
-  projectName: string;
+  setPlaying: (value: boolean) => void;
 }
 
 const MusicPlayView: React.FunctionComponent<MusicPlayViewProps> = ({
-  isPlaying,
-  onPlay,
-  onStop,
-  projectName,
+  setPlaying,
 }) => {
+  const {isPlaying, lastMeasure, currentPlayheadPosition} = useAppSelector(
+    state => state.music
+  );
+  const progressValue =
+    lastMeasure === 1
+      ? 0
+      : ((currentPlayheadPosition - 1) / (lastMeasure - 1)) * 100;
```

Comment:
Might be good to put this whole calculation inside the selector so that way the `progressValue` is automatically memoized and this component only updates when that value changes, rather than any time the position updates? Actually speaking it may end up being the same amount of updates, but for example if we were to round this number, then it would be helpful since we'd only update when the actual value changed. Ex:
```
const progressValue = useAppSelector(state => {
  const {currentPlayheadPosition, lastMeasure} = state.music;
  // ... calculate value
  return progressValue;
}